tests/queries/0_stateless/03142_skip_ANSI_in_UTF8_compute_width.sql

```diff
@@ -0,0 +1 @@
+SELECT format('\x1b[38;2;{0};{1};{2}m█\x1b[0m', 255, 128, 0) AS x FORMAT Pretty;
```

Comment:
```suggestion
SELECT format('\x1b[38;2;{0};{1};{2}m█\x1b[0m', 255, 128, 0) AS x FORMAT Pretty;

```

Reply:
@antaljanosbenjamin the change looks identical, apart from an empty line after the query - just curious if there is some specific reason for this?

Reply:
It is a [good practice](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) in general to have every text file end with an empty line.